API.md

```diff
@@ -401,6 +401,62 @@ After first iteration:
 */
 ```
 
+## `client.proposeStorageDeal`
+
+> Propose a storage deal with a storage miner
+
+### `client.proposeStorageDeal(miner, cid, askID, time, allowDuplicates, [options])`
+
+#### Parameters
+
+| Name | Type | Description |
+|------|------|-------------|
+| miner | `String` | Address of miner to send storage proposal |
+| cid | `String` | CID of the data to be stored |
+| id | `String` | ID of ask for which to propose a deal |
+| time | `String` | Time in blocks (about 30 seconds per block) to store data. For example, storing for 1 day (2 blocks/min * 60 min/hr * 24 hr/day) = 2880 blocks. |
+| allowDuplicates | `Boolean` | Allows duplicate proposals to be created. Unless this flag is set, you will not be able to make more than one deal per piece per miner. This protection exists to prevent erroneous duplicate deals. This parameter is not required. |
```

Comment:
This should be `options.allowDuplicates`

Reply:
Great! Thanks for the clarity.